src/darwin/Framework/CHIP/Matter.h

```diff
@@ -68,5 +68,6 @@
 #import <Matter/MTRStorage.h>
 #import <Matter/MTRStructsObjc.h>
 #import <Matter/MTRThreadOperationalDataset.h>
+#import <Matter/MTRUtilities.h>
```

Comment:
We were missing an import, but we were extern-ing it. Adding it. I'll fix this file and others with a mass xcconfig fix.